notification_context.md

```diff
@@ -17,6 +17,8 @@ eventType | string | Notification.Message.Created
 data | hash |
 &nbsp;&nbsp;externalReference | string | external, unique case identifier; **required**
 &nbsp;&nbsp;masterdataReference | string | reference of a property / building / unit; **required**
+&nbsp;&nbsp;categoryCode | string | optional for the category code - overrides the default looked-up using the remote system_name (useful for systems that need to create multiple types of notifications)
```

Comment:
Not sure I understand this sentence, but I would propose to reformulate it a little bit.

`optional category code to override the default looked-up using the remote system_name. Useful for systems that need to create multiple types of notifications.`

Now, you are talking about a _default looked-up using the remote system_name_.
To me, that's the unclear part.

Reply:
Thanks for you comment - I have updated the text.  Can you let me know if it is now clearer?

Reply:
I think the term _look-up_ is unclear in that context. 
What's this _look-up_? Is that related to the data the remote system will retrieve through GraphQL?

Reply:
would _resolve_ be clearer? 

or even just:

```
&nbsp;&nbsp;categoryCode | string | optional for the category code - this value must match a 'code' within the 'notification_kategorie_cd' of the CodeTabelle (Garaio REM staff can clarify or created the necessary codes as needed).  Non-matching values will fall back to the default based on the sending system's hostname.  This is useful for remote systems that need to provide multiple types of notifications.

&nbsp;&nbsp;recipientUsername | string | optional recipient username (within GREM) to override the default recipient.  Blank or non-matching values will fallback to the default based on the given 'masterdataReference'.
```